packages/ariakit/src/focus-trap/__examples__/focus-trap/test.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  click,
+  focus,
+  getByLabelText,
+  getByRole,
+  getByText,
+  press,
+  render,
+} from "ariakit-test-utils";
+import Example from ".";
+
+test("correctly traps focus", async () => {
+  // Focus trap is on by default in the Example
+  render(
+    <div>
+      <Example />
+      <div tabIndex={0}>Outside</div>
+    </div>
```

Comment:
Nice! 👍  We usually render just `<Example />`, but these extra elements are handy here. Do you think we should add another element before so we can test shift+tab as well?

Reply:
Oh yea! Let's do that 💪